### PR TITLE
feat(serverless): added nats cli doc

### PR DESCRIPTION
**Pull request by RoRoJ**
### Description
Add doc on NATS CLI.

### CODE REVIEW

File: serverless/messaging/api-cli/nats-cli.mdx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  paragraph: This page explains how to use the NATS CLI with Scaleway Messaging and Queuing
+tags: messaging nats cli
+categories: 
+  - messaging
+dates:
+  validation: 2023-01-20
+  posted: 2023-01-20
+---
+
+The NATS CLI (`nats`) is the official NATS tool for managing your NATS resources. It aloows you to simply create and manage your streams, consumers and more.
```
Comment (nedaehm) [inline serverless/messaging/api-cli/nats-cli.mdx:16]:
```suggestion
The NATS CLI (`nats`) is the official NATS tool for managing your NATS resources. It allows you to simply create and manage your streams, consumers and more.
```

File: serverless/messaging/api-cli/nats-cli.mdx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Enter the following command to select the `scaleway` context for use:
+
+```
+nats context select scaleway
+```
+
+## Create a stream
+
+To create a [stream](/serverless/messaging/concepts/#stream), use the command `nats stream add` and follow the CLI guidelines.
```
Comment (nedaehm) [inline serverless/messaging/api-cli/nats-cli.mdx:58]:
```suggestion
To create a [stream](/serverless/messaging/concepts/#stream/), use the command `nats stream add` and follow the CLI guidelines.
```

File: serverless/messaging/api-cli/nats-cli.mdx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+<Message type="important">
+  Bear in mind that:
+  - Scaleway Messaging and Queuing does not support in-memory streams: choose `File` as storage backend.
+  - Some [system limits](/serverless/messaging/reference-content/limitations/) may apply by default.
+  - If you choose a `Retention Policy` other than `Work Queue` you will be billed for the messages stored and retained.
+  - Choosing three replicas has an implact on:
```
Comment (nedaehm) [inline serverless/messaging/api-cli/nats-cli.mdx:65]:
```suggestion
  - Choosing three replicas has an impact on:
```

**Review by bene2k1 — APPROVED** [review]

File: serverless/messaging/api-cli/nats-cli.mdx (hunk 2 above)
Comment (RoRoJ) [reply serverless/messaging/api-cli/nats-cli.mdx:58]: I think the slash has to be in the middle (ie it has to be concepts/#<name-of-concept) but not at the end else it doesn't take you to the anchored concept, just the page

**Review by ofranc — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

File: serverless/messaging/reference-content/nats-overview.mdx [hunk 4]
```diff
@@ -35,7 +35,7 @@ All further actions related to publishing, processing and managing messages, sub
 
 ### NATS CLI 
 
-The [NATS CLI](https://docs.nats.io/using-nats/nats-tools/nats_cli) is a great tool for administrating or testing your subjects and streams. 
+The [NATS CLI](https://docs.nats.io/using-nats/nats-tools/nats_cli) is a great tool for administrating or testing your subjects and streams. See our [dedicated documentation](/serverless/messaging/api-cli/nats-cli) to help get started with the NATS CLI.
```
Comment (ofranc) [inline serverless/messaging/reference-content/nats-overview.mdx:38]:
```suggestion
The [NATS CLI](https://docs.nats.io/using-nats/nats-tools/nats_cli/) is a great tool for administrating or testing your subjects and streams. See our [dedicated documentation](/serverless/messaging/api-cli/nats-cli/) to help get started with the NATS CLI.
```

File: serverless/messaging/api-cli/nats-cli.mdx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+Enter the following command to select the `scaleway` context for use:
+
+```
+nats context select scaleway
+```
+
+## Create a stream
+
+To create a [stream](/serverless/messaging/concepts/#stream), use the command `nats stream add` and follow the CLI guidelines.
```
Comment (ofranc) [inline serverless/messaging/api-cli/nats-cli.mdx:58]:
```suggestion
To create a [stream](/serverless/messaging/concepts/#stream/), use the command `nats stream add` and follow the CLI guidelines.
```

File: serverless/messaging/api-cli/nats-cli.mdx [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+## Define a context
+
+To simplify your interactions with NATS hosted on Scaleway Messaging and Queuing, we recommend that you use [contexts](https://docs.nats.io/using-nats/nats-tools/nats_cli#nats-contexts). A context is a named configuration that stores the settings (such as credentials, URLs and certificates) required to connect to a NATS namespace. By creating a context, you won't need to specify your server and credentials with each new request.
+
+### Create a context
+
+The example below creates a context named `scaleway`.
+- Replace `{Scaleway NATS URL}` with the URL of your NATS namespace (find this in the console on your [Namespaces dashboard](https://console.scaleway.com/messaging/namespaces)).
+- Replace `{path to your creds file}` with the path to the location where you [downloaded your `.creds` file](https://www.scaleway.com/en/docs/serverless/messaging/how-to/create-credentials/#how-to-create-credentials-for-a-nats-namespace).
```
Comment (ofranc) [inline serverless/messaging/api-cli/nats-cli.mdx:42]:
```suggestion
- Replace `{path to your creds file}` with the path to the location where you [downloaded your `.creds` file](https://www.scaleway.com/en/docs/serverless/messaging/how-to/create-credentials/#how-to-create-credentials-for-a-nats-namespace/).
```

File: serverless/messaging/api-cli/nats-cli.mdx [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)  
+  - You have [created a NATS Messaging namespace](/serverless/messaging/how-to/create-namespace/)
+  - You have [generated credentials](/serverless/messaging/how-to/create-credentials/) for your NATS Messaging and Queuing namespace and downloaded them to your machine.
+</Message>
+
+## Installing the NATS CLI
+
+Follow instructions from the [official NATS documentation](https://docs.nats.io/using-nats/nats-tools/nats_cli#installing-nats). The installation process may differ depending on your OS.
```
Comment (ofranc) [inline serverless/messaging/api-cli/nats-cli.mdx:32]:
```suggestion
Follow instructions from the [official NATS documentation](https://docs.nats.io/using-nats/nats-tools/nats_cli#installing-nats/). The installation process may differ depending on your OS.
```

File: serverless/messaging/api-cli/nats-cli.mdx [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+The NATS CLI (`nats`) is the official NATS tool for managing your NATS resources. It allows you to simply create and manage your streams, consumers and more.
+
+Check out the official [NATS CLI documentation](https://docs.nats.io/using-nats/nats-tools/nats_cli) for installation instructions, examples and more.
+
+This page shows you how to get started with some basic actions via the NATS CLI.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
```
Comment (ofranc) [inline serverless/messaging/api-cli/nats-cli.mdx:25]:
```suggestion
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)  
```

File: serverless/messaging/api-cli/nats-cli.mdx [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+categories: 
+  - messaging
+dates:
+  validation: 2023-01-20
+  posted: 2023-01-20
+---
+
+The NATS CLI (`nats`) is the official NATS tool for managing your NATS resources. It allows you to simply create and manage your streams, consumers and more.
+
+Check out the official [NATS CLI documentation](https://docs.nats.io/using-nats/nats-tools/nats_cli) for installation instructions, examples and more.
```
Comment (ofranc) [inline serverless/messaging/api-cli/nats-cli.mdx:18]:
```suggestion
Check out the official [NATS CLI documentation](https://docs.nats.io/using-nats/nats-tools/nats_cli/) for installation instructions, examples and more.
```

**Review by ofranc — APPROVED** [review]

**Pull request merged**